iosApp/iosApp/ComponentViews/RouteHeader.swift

```diff
@@ -28,5 +28,7 @@ struct RouteHeader<Content: View>: View {
             modeIcon: routeIcon(route),
             rightContent: rightContent
         )
+        .accessibilityAddTraits(.isHeader)
+        .accessibilityHeading(.h2)
```

Comment:
I heard "Red Line. Heading level 2. Star route. Heading level 2". Maybe applying the heading to the route text only or using `accessibilityElement(combine)` would help?

Reply:
83b87641cf611e7c979dcbfeaac139278c05ed8f